content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -21,33 +21,60 @@ further_reading:
 
 - The .NET Tracer library for Datadog is open source. For more information, see the [.NET Tracer repository][1].
 
+## Runtime support policy for .NET Core APM
+
+Datadog APM for .NET Core is built upon dependencies defined in specific versions of the host operating system, .NET Core runtime, certain .NET Core libraries, and the Datadog Agent/API. When these versions are no longer supported by their maintainers, Datadog APM for .NET Core limits its support for these as well.
+
+### Levels of support
+
+delner marked this conversation as resolved.
+
+| **Level**                                              | **Support provided**                                                                                                                                                          |
+|--------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      |  No implementation. [Contact our customer support team for special requests.][2]                                                             |
+| <span id="support-beta">Beta</span>                    |  Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
+| <span id="support-ga">General Availability (GA)</span> |  Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
+| <span id="support-maintenance">Maintenance</span>      |  Full implementation of existing features. Does not receive new features. Support for bug & security fixes only.                                                              |
+| <span id="support-legacy">Legacy</span>                |  Legacy implementation. May have limited function, but no maintenance provided. [Contact our customer support team for special requests.][2] |
+| <span id="support-eol">End-of-life (EOL)</span>        |  No support.                                                                                                                                                                  |
+
+### Package versioning
+
+Datadog APM for .NET Core practices [semantic versioning][3].
+As this relates to downgrading runtime support, it implies:
+
+  - **Major version updates** (e.g. `1.0.0` to `2.0.0`) may change support for any runtime from [Beta](#support-beta)/[GA](#support-ga) to [Maintenance](#support-maintenance)/[Legacy](#support-legacy)/[EOL](#support-eol).
+  - **Minor version updates** (e.g. `1.0.0` to `1.1.0`) will not change support for any runtime.
```

Comment:
We could potentially _add_ support for a runtime here

Reply:
All this work (see the original RFC) was specifically about downgrading or dropping support. The line above says

> As this relates to downgrading runtime support...

I wonder if we need to make this more clear, or just explicitly mention when support can be added/upgraded. 🤔